drivers/nrf_radio_802154/platform/random/nrf_802154_random_zephyr.c

```diff
@@ -75,7 +75,7 @@ void nrf_802154_random_init(void)
     struct device *dev;
     int err;
 
-    dev = device_get_binding(CONFIG_ENTROPY_NAME);
+    dev = device_get_binding(DT_ENTROPY_NAME);
```

Comment:
Is the nrf_radio_802154 driver only useful on a NRF SoC?  If so, I wonder if this would better to use a DT_LABEL(DT_INST(0, nordic_nrf_rng)) here.

Reply:
Right. No need to depend on that generic symbol in an nRF specific code. I'll modify it.
Probably the same is applicable to `subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c` in https://github.com/zephyrproject-rtos/zephyr/pull/24127 (see [this change](https://github.com/zephyrproject-rtos/zephyr/pull/24127/files#diff-af287d2745e308edf9604e7be8b2f7b8)).